<?xml version="1.0" encoding="UTF-8"?><magicdraw><comment classType="Comment" humanType="Comment" icon="index_files/icon_comment_2051026490.jpg" id="_10_5_622020d_1132928316060_257237_10462"><annotatedElement humanName="Annotated Element" mode="s,e"><class classType="Class" humanType="Class" icon="index_files/icon_class_1555793073.jpg" name="CredentialNotFoundException" refid="_10_5_622020d_1132928316050_569743_10460"/></annotatedElement><body humanName="Body" mode="s,e">Signals that a credential was not found.

&lt;p&gt; This exception may be thrown by a LoginModule if it is unable
to locate a credential necessary to perform authentication.

@version 1.2, 12/19/03
@since 1.5
</body><documentation humanName="Documentation"/><owner classType="Class" humanName="Owner" humanType="Class" icon="index_files/icon_class_1555793073.jpg" mode="s,e" name="CredentialNotFoundException" refid="_10_5_622020d_1132928316050_569743_10460"/><owningElement classType="Class" humanName="Owning Element" humanType="Class" icon="index_files/icon_class_1555793073.jpg" name="CredentialNotFoundException" refid="_10_5_622020d_1132928316050_569743_10460"/><TO_DO humanName="To Do" mode="s,e"/></comment></magicdraw>